apps/user-profile/package.json

```diff
@@ -14,6 +14,7 @@
     "autosave": "^1.0.3",
     "axios": "^0.27.2",
     "constate": "^3.3.2",
+    "framer-motion": "6.2.4",
```

Comment:
We use framer-motion version 6.5.1 throughout the monorepo, it's better be consistent with the rest of the apps when it comes to deps versions :)